src/common/Backend.ts

```diff
@@ -182,6 +182,9 @@ export class Backend {
                 if (error.response?.data) {
                     return error.response.data as T;
                 }
+                if (axios.isCancel(error)) {
+                    return {} as T;
```

Comment:
What even is this?

Reply:
As is, the function has to return T. What would you recommend doing instead?

Reply:
Are.... Are you joking? The function has to return a `T`, so you just cast some random value to `T`?

This PR is supposed to fix a bug, not add another one that's harder to detect.

Reply:
Ok, yeah